Extensions/ValuesOfMultipleObjects.json

```
@@ -0,0 +1,1796 @@
+            "textR": 0
+          },
+          "comment": "Cycle through units",
+          "comment2": ""
+        },
+        {
+          "disabled": false,
+          "folded": false,
+          "type": "BuiltinCommonInstructions::ForEach",
+          "object": "Object",
+          "conditions": [],
+          "actions": [],
+          "events": [
+            {
+              "disabled": false,
+              "folded": false,
+              "type": "BuiltinCommonInstructions::Standard",
+              "conditions": [
+                {
+                  "type": {
+                    "inverted": false,
+                    "value": "CenterY"
+                  },
+                  "parameters": [
+                    "Object",
+                    ">",
+                    "Variable(__ValuesOfObjects.GroupMaxY)"
+                  ],
+                  "subInstructions": []
+                }
+              ],
+              "actions": [
+                {
+                  "type": {
+                    "inverted": false,
+                    "value": "ModVarScene"
+                  },
+                  "parameters": [
+                    "__ValuesOfObjects.GroupMaxY",
+                    "=",
+                    "Object.CenterY()"
+                  ],
+                  "subInstructions": []
+                }
+              ],
+              "events": []
+            }
+          ]
+        },
+        {
+          "disabled": false,
+          "folded": false,
+          "type": "BuiltinCommonInstructions::Comment",
+          "color": {
+            "b": 109,
+            "g": 230,
+            "r": 255,
+            "textB": 0,
+            "textG": 0,
+            "textR": 0
+          },
+          "comment": "Return the value",
+          "comment2": ""
+        },
+        {
+          "disabled": false,
+          "folded": false,
+          "type": "BuiltinCommonInstructions::Standard",
+          "conditions": [],
+          "actions": [
+            {
+              "type": {
+                "inverted": false,
+                "value": "SetReturnNumber"
+              },
+              "parameters": [
+                "Variable(__ValuesOfObjects.GroupMaxY)"
+              ],
+              "subInstructions": []
+            }
+          ],
+          "events": []
+        }
+      ],
+      "parameters": [
+        {
+          "codeOnly": false,
+          "defaultValue": "",
+          "description": "Objects",
+          "longDescription": "Objects that will be used to calculate the value",
+          "name": "Object",
+          "optional": false,
+          "supplementaryInformation": "",
+          "type": "objectList"
+        }
+      ],
+      "objectGroups": []
+    },
+    {
+      "description": "X center point of picked object instances (middle)",
+      "fullName": "X center point of picked object instances (middle)",
+      "functionType": "Expression",
+      "name": "CenterOfObjectsX_Middle",
```

```suggestion
      "name": "CenterOfObjectsX",
```

I think that the `_Middle` sounds not very friendly to a user, and Center and Middle should have the same meaning to 99.9% of the users, so I'd leave it out here.

---

Do you like these names better?

Option 1: "CenterOfObjectsX" and "WeightedCenterOfObjectsX"
or 
Option 2: "AbsoluteCenterOfObjectsX" and "WeightedCenterOfObjectsX"

BTW, I think it is important to keep both of these because camera functionality wants absolute values, while flocking needs the weighted version.

---

Don't get me wrong i was talking about leaving the word `Middle` out because the word `Center` already carries it's meaning, not to remove the function

---

🤔 maybe option 2 since it makes the difference more clear